vnext/Microsoft.ReactNative/Modules/TimingModule.h

```diff
@@ -11,6 +11,7 @@
 #include <vector>
 
 #include <winrt/Windows.Foundation.h>
+#include <winrt/Windows.System.h>
```

Comment:
winui3 must use Microsoft::DispatcherQueue instead from `winrt/Microsoft.System.h`

Reply:
@asklar - are there plans to set up a CI build for WinUI3? It would be nice to get a failing signal for this.

Reply:
Hmm, I thought we already did - I remember CI failing due to incorrect namespaces, but then it passes for this PR, so idk

Reply:
we do have CI builds for WinUI 3, this will compile as it still valid code, but will crash at runtime :)